lib/active_admin_versioning/active_admin/resource_controller.rb

```diff
@@ -18,6 +18,10 @@ def show
       protected
 
       def user_for_paper_trail
+        ActiveAdminVersioning.configuration.user_for_paper_trail ? self.send(ActiveAdminVersioning.configuration.user_for_paper_trail) : super
+      end
```

Comment:
`super` calls:

> ```ruby
> # Returns the user who is responsible for any changes that occur.
> # By default this calls `current_user` and returns the result.
> #
> # Override this method in your controller to call a different
> # method, e.g. `current_person`, or anything you like.
> def user_for_paper_trail
>   return unless defined?(current_user)
>   ActiveSupport::VERSION::MAJOR >= 4 ? current_user.try!(:id) : current_user.try(:id)
> rescue NoMethodError
>   current_user
> end
> ```

https://github.com/airblade/paper_trail/blob/5-stable/lib/paper_trail/frameworks/rails/controller.rb#L29

hmmm..., I think it's best way to override this method, too. 

I might implement it like this (another approach):

```ruby
def user_for_paper_trail
  respond_to?(current_user_method) ? send(current_user_method).id : t('views.version.unknown_user')
end

private

def current_user_method
  ActiveAdmin.application.namespaces[:admin].current_user_method
end
```